conf/test_full.config

```diff
@@ -72,6 +72,9 @@ params {
     run_kmcp                               = true
     kmcp_save_search                       = true
 
+
+    run_sylph				                = true
+    sylph_taxonomy                         = '/home/proj/development/microbial/metagenomics/sofia_workspace/full_test_data/meslier2022/sylph/sylph_taxonomy.tsv.gz'
```

Comment:
This needs to be updated to a remote URL :grimacing:

Reply:
Updated :D